site/_includes/partials/script.js

```diff
@@ -38,6 +38,15 @@
     '{{ analytics.dimensions.TRACKING_VERSION }}',
     '{{ analytics.TRACKING_VERSION }}'
   );
+  try {
+    ga(
+      'set',
+      '{{ analytics.dimensions.NAVIGATION_TYPE }}',
+      performance.getEntriesByType('navigation')[0].type
```

Comment:
Since we're already using the navigation type reported by the `web-vitals` library, to be consistent with that let's standardize on using kebab-case.

```suggestion
      performance.getEntriesByType('navigation')[0].type.replace(/_/g, '-')
```

Reply:
I noticed it was inconsistent on web.dev and wondered should we change bfcache one! But happy to change this instead.

Reply:
Yeah, I think we should probably also change web.dev...